### PR TITLE
test(fix): do not skip backup ITs

### PR DESCRIPTION
**Pull request by kolea2**
Cluster ID was null which was causing the tests to skip. Additionally did some cleanup on test structure

### CODE REVIEW

File: google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java [hunk 1]
```diff
@@ -56,11 +64,11 @@
 
 @RunWith(JUnit4.class)
 public class BigtableBackupIT {
+  @ClassRule public static TestEnvRule testEnvRule = new TestEnvRule();
```
Comment (igorbernstein2) [inline google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java:67]: I think this integration test is in a weird state: it was written in fork where TestEnvRule didnt exist and uses its own property names (ie. bigtable.adminendpoint vs bigtable.admin-endpoint. So this test should either migrate to the TestEnvRule now or we should stick to the bespoke property names

Comment (kolea2) [reply google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java:67]: I'll give migrating a try

Comment (kolea2) [reply google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java:63]: @igorbernstein2 switched over to TestEnvRule, PTAL

**Review by igorbernstein2 — APPROVED** [review]
lgtm

**Pull request merged**